python_modules/libraries/dagster-dbt/dagster_dbt/dagster_dbt_translator.py

```diff
@@ -520,6 +521,42 @@ def get_automation_condition(self, dbt_resource_props: Mapping[str, Any]) -> Opt
             auto_materialize_policy.to_automation_condition() if auto_materialize_policy else None
         )
 
+    @public
+    @experimental(emit_runtime_warning=False)
+    def get_partitions_def(
```

Comment:
Can we add a comment here and on the callsite to mention that this is a temporary measure needed for internal usage? This should be accessed as `get_asset_spec(...).partitions_def` when we have `get_asset_spec`